spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe '#UserIndex', type: :feature do
+  before(:each) do
+    @users = [
+      @user1 = User.create(
+        name: 'Derrick',
+        photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
+        bio: 'Fullstack Web developer from Uganda',
+        posts_counter: 9
+      ),
+      @user2 = User.create(
+        name: 'Lilly',
+        photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
+        bio: 'Teacher from Poland.', posts_counter: 1
+      )
+    ]
+
+    @posts = [
+      @post1 = Post.create(
```

Comment:
* As the previous CR pointed out I still see some failing tests, Kindly fix them 🙏 
![image](https://user-images.githubusercontent.com/52497246/220840558-7ca26be7-4e33-4ba8-bff6-22cd890c61cb.png)